### PR TITLE
added rtx 3050 mobile and also added 6GB to DEFAULT_MEMORY_OPTIONS

### PR DESCRIPTION
**Pull request by Om-Alve**
Added info about RTX 3050 mobile 6GB and also added 6GB to the DEFAULT_MEMORY_OPTIONS list

### CODE REVIEW

**Comment by coyotte508** [comment]
Thank you for the PR!

The `DEFAULT_MEMORY_OPTIONS` is used for CPU RAM, can you make a separate PR for this change? I'm not sure 6GB RAM is very common with latops

**Comment by Om-Alve** [comment]
> The `DEFAULT_MEMORY_OPTIONS` is used for CPU RAM, can you make a separate PR for this change? I'm not sure 6GB RAM is very common with laptops

Ah, I'm sorry I thought that they were the default values for VRAM, should I undo that change in this PR?

File: packages/tasks/src/hardware.ts [hunk 1]
```diff
@@ -128,6 +128,10 @@ export const SKUS = {
 				tflops: 12.74,
 				memory: [12, 8],
 			},
+			"RTX 3050 Mobile":{
+				tflops: 7.639,
+				memory:[6],
```
Comment (pcuenca) [inline packages/tasks/src/hardware.ts:133]: The first version was slower and had 4 GB: https://www.techpowerup.com/gpu-specs/geforce-rtx-3050-mobile.c3788

Let's keep this one for now (Numbers match this reference: https://www.techpowerup.com/gpu-specs/geforce-rtx-3050-mobile-refresh-6-gb.c3971)

**Review by pcuenca — APPROVED** [review]
Thanks a lot!

File: packages/tasks/src/hardware.ts [hunk 2]
```diff
@@ -152,6 +152,10 @@ export const SKUS = {
...
+			}
```
Comment (pcuenca) [inline packages/tasks/src/hardware.ts:158]:
```suggestion
			},
```

File: packages/tasks/src/hardware.ts [hunk 3]
```diff
@@ -152,6 +152,10 @@ export const SKUS = {
 				tflops: 12.74,
 				memory: [12, 8],
 			},
+			"RTX 3050 Mobile":{
+				tflops: 7.639,
+				memory:[6],
```
Comment (pcuenca) [inline packages/tasks/src/hardware.ts:157]:
```suggestion
				memory: [6],
```

**Pull request merged**